examples/ios/objc/Draw/Views/SwatchesView.m

```diff
@@ -0,0 +1,138 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2014 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import "SwatchesView.h"
+
+static CGFloat kSwatchButtonHeightPhone = 85.0f;
+static CGFloat kSwatchButtonWidthPhone = 30.0f;
+
+static CGFloat kSwatchButtonHeightPad= 166.0f;
+static CGFloat kSwatchButtonWidthPad = 57.0f;
+
+static CGFloat kSwatchPencilPadding = 1.0f;
+
+@interface SwatchesView()
+
+@property (nonatomic, strong) NSDictionary *colors;
+@property (nonatomic, strong) NSArray *colorButtons;
+
+- (void)setupButtons;
+- (void)buttonTapped:(id)sender;
++ (CGSize)sizeForDevice;
+- (void)updateContentInset;
+
+@end
+
+@implementation SwatchesView
+
+- (instancetype)initWithFrame:(CGRect)frame
+{
+    frame.size.height = [SwatchesView sizeForDevice].height;
+    if (self = [super initWithFrame:frame]) {
+        [self setupButtons];
+    }
+    
+    return self;
+}
+
+- (void)setupButtons
+{
+    self.colors = [UIColor realmColors];
+
+    NSMutableArray *buttons = [NSMutableArray arrayWithCapacity:self.colors.count];    
+    NSInteger tag = 0;
+    for (NSString *color in self.colors.allKeys) {
+        UIImage *pencilImage = [[UIImage imageNamed:color] imageWithRenderingMode:UIImageRenderingModeAlwaysOriginal];
```

Comment:
why is `-[UIImage imageWithRenderingMode:UIImageRenderingModeAlwaysOriginal]` useful in this case? Why wouldn't just `UIImage *pencilImage = [UIImage imageNamed:color];` be sufficient here?